docs/guides/connecting_to_your_data/cloud/s3/components_spark/_test_your_new_datasource.md

```diff
@@ -0,0 +1,46 @@
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+
+Verify your new <TechnicalTag tag="datasource" text="Datasource" /> by loading data from it into a <TechnicalTag tag="validator" text="Validator" /> using a <TechnicalTag tag="batch_request" text="Batch Request" />.
+
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Specify an S3 path to single CSV', value:'runtime_batch_request'},
+  {label: 'Specify a data_asset_name', value:'batch_request'},
+  ]}>
+
+<TabItem value="runtime_batch_request">
+
+Add the S3 path to your CSV in the `path` key under `runtime_parameters` in your `RuntimeBatchRequest`.
+
+:::tip
+The path you will want to use is your S3 URI, not the URL.
+:::
+
+```python file=../../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/s3/spark/inferred_and_runtime_yaml_example.py#L57-L63
+```
+
+Then load data into the `Validator`.
+
+```python file=../../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/s3/spark/inferred_and_runtime_yaml_example.py#L71-L77
+```
+
+</TabItem>
+
+<TabItem value="batch_request">
+
+Add the name of the <TechnicalTag tag="data_asset" text="Data Asset" /> to the `data_asset_name` in your `BatchRequest`.
+
+```python file=../../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/s3/spark/inferred_and_runtime_yaml_example.py#L83-L88
+```
+
+Then load data into the `Validator`.
```

Comment:
Hi Rachel, I got to this part and am currently getting the following error: 

```
Py4JJavaError: An error occurred while calling o124.csv.
: java.lang.RuntimeException: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
```

* I'm beginning to look into it ([Helpful StackOverflow](https://stackoverflow.com/questions/58415928/spark-s3-error-java-lang-classnotfoundexception-class-org-apache-hadoop-f)) and I think there may be some additional set up needed on my machine in order for Spark to read from S3.  As a consequence of this, I currently don't think it it's going to be a 1:1 conversion from the Pandas to Spark document, but some additional explanation (and set up) will be needed.  I'll update to this thread once I have more information